.github/workflows/semantic.yml

```diff
@@ -65,10 +127,16 @@ jobs:
       - name: Build warplib
         run: yarn warplib
 
+      - name: Setup gcc-11 Environment
+        run: |
+          sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
+          sudo apt install -y gcc-11 gcc-11-multilib g++-11 g++-11-multilib
+          gcc-11 --version
+
       - name: Test code execution
         run: |
-          yarn testnet:benchmark &
-          node build/autoRunSemanticTests.js
+          yarn testnet:benchmark_semantic &
+          SM_BATCH=3 node build/autoRunSemanticTests.js
```

Comment:
No need to copy-paste code. It is possible to reuse code via custom actions, reusable workflows or matrices. In this case I'd suggest using matrix

Reply:
good point! done